CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 2.28.0 (November 23, 2023)
 FEATURES:
 * resource/sumologic_monitor: Added support for setting `time_zone` at Monitor level for notifications content (GH-586)
+* resource/sumologic_muting_schedule: Added group supporting for muting schedule (GH-593)
```

Comment:
2.28.0 is already out. This release note should go under "unreleased" section.

Reply:
updated.